src/capstone2llvmir/capstone2llvmir_impl.h

```diff
@@ -325,24 +330,109 @@ class Capstone2LlvmIrTranslator_impl : virtual public Capstone2LlvmIrTranslator
+
+		/**
+		 * Creates LLVM load from LLVM value representing
+		 * operand of instruction ci on index idx. User
+		 * of this method may specify type to which will be
+		 * loaded value converted and method of the conversion.
+		 *
+		 * @param ci       Instruction of which operand will be loaded.
+		 * @param irb      LLVM IR Builder required for IR modifications.
+		 * @param idx      Operand index.
+		 * @param loadType Type of loaded value. (not relevant if nullptr)
+		 * @param dstType  Desired type of loaded value (not changed if nullptr).
+		 * @param ct       Used conversion. Defaultly NOTHING as "do not convert".
+		 */
+		llvm::Value* loadOp(
+				CInsn* ci,
+				llvm::IRBuilder<>& irb,
+				std::size_t idx,
+				llvm::Type* loadType = nullptr,
+				llvm::Type* dstType = nullptr,
+				eOpConv ct = eOpConv::NOTHING);
+
+		/**
+		 * Create LLVM loads for LLVM values representing last N operands
+		 * (opCnt) of specified instruction. If strict check is set, this
+		 * method will check wheater number of operands of the instructions
+		 * is equal to the "opCnt". If conversion type is set to NOTHING
+		 * no conversion will happen and each operand may have different
+		 * size and type.
+		 *
+		 * This method was created to be used in internal load
+		 * methods. Usage of adequate loadOp(Binary|Ternary|...)
+		 * is preffered.
+		 *
+		 * @param ci	Instruction of which operands will be loaded.
+		 * @param irb	LLVM IR Builder required for IR modifications.
+		 * @param opCnt	Number of operands that will be loaded.
+		 * @param strictCheck	If set to true opCnt will be equal as number of operands. Otherwise will load N last operands.
+		 * @param loadType	Type of loaded value. (not relevant if nullptr)
+		 * @param dstType	Desired type of loaded value (not changed if nullptr).
+		 * @param ct		Used conversion. Defaultly NOTHING as "do not convert".
+		 */
+		std::vector<llvm::Value*> _loadOps(
+				CInsn* ci,
+				llvm::IRBuilder<>& irb,
+				std::size_t opCnt,
+				bool strictCheck = true,
+				llvm::Type* loadType = nullptr,
+				llvm::Type* dstType = nullptr,
+				eOpConv ct = eOpConv::NOTHING);
+
+		/**
+		 * Similiar functionality as `_loadOps` but used conversion is determined
+		 * by type of first loaded operand. This means that if first operand
+		 * is of integer type then `ict` convertion will be used on all other opernads.
+		 * If first perand is floting point type then used convertion will be `fct`.
+		 *
+		 * @param ci
+		 * @param irb
+		 * @param opCnt
+		 * @param strictCheck
+		 * @param ict	Integer convertion type.
+		 * @param fct	Floting point convertion type.
+		 */
+		std::vector<llvm::Value*> _loadOpsUniversal(
+				CInsn* ci,
+				llvm::IRBuilder<>& irb,
+				std::size_t opCnt,
+				bool strictCheck = true,
+				eOpConv ict = eOpConv::SEXT_TRUNC_OR_BITCAST,
+				eOpConv fct = eOpConv::FPCAST_OR_BITCAST);
 
-		llvm::Value* loadOpUnary(
+		llvm::Value* loadOpsUnary(
```

Comment:
Why `loadOpsUnary()` with `s`? There was `loadOpUnary()` before and there are `loadOpBinary()` now (with no `s`). Is this intentional? If so, what is the reason? Or is it a typo? If so, fix it back.

Reply:
Oh, I forgot to change this one back. Originally I wanted to create new conversion methods, and this would help me identify all their usages. After a few methods I changed my mind, and I guess this was in another commit when I reverted the change. It is now a typo; I will change it back.